react/week5/my-app/src/components/GithubUserSearch.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useEffect } from "react";
+import { Card, Icon, Image } from "semantic-ui-react";
+
+const GithubUserSearch = () => {
+  const [userInput, setUserInput] = useState("");
+  const [data, setData] = useState("");
+  const [loading, setLoading] = useState(false);
+  const [currentUserDetails, setCurrentUserDetails] = useState("");
+  const [error, setError] = useState(null);
+
+  const userClickHandler = (login) => {
```

Comment:
This function is too large, its doing too much. When something is clicked i would expect some state to be set and then depending on that state something will be rendered. You are creating the markup in the click handler. The markup i would expect to be in teh render function at line 93.